CMakeLists.txt

```diff
@@ -1,8 +1,9 @@
 cmake_minimum_required(VERSION 2.8)
 project(optional)
 
-add_library(optional INTERFACE)
-target_include_directories(optional
+add_library(akrzemi1_optional INTERFACE)
+add_library(optional ALIAS akrzemi1_optional)
```

Comment:
can we add alias `akrzemi1::optional` and have downstream repositories refer to that, avoids confusion and cmake knows its linking against a cmake target